apps/src/lib/ui/accounts/ManageLinkedAccounts.jsx

```diff
@@ -199,6 +204,10 @@ class OauthConnection extends React.Component {
     }
   };
 
+  shouldDisableConnectButton = userAge => {
+    return !!(window.CPA_EXPERIENCE && userAge < 13);
```

Comment:
I was thinking this should be limited to US students. Could you double check with Ali if we are okay with applying this to all U13?

Reply:
FYI, just spoke with Ali and we also need to exclude Clever from any age gates. Working on doing that.